tests/capture/autograph/test_autograph.py

```diff
@@ -349,6 +349,34 @@ def circ():
         ):
             _ = qml.capture.make_plxpr(circ, autograph=True)()
 
+    @pytest.mark.parametrize(
+        "func1, func2",
+        [
+            (qml.adjoint, qml.ctrl),
+            (qml.ctrl, qml.adjoint),
+            (qml.adjoint, qml.adjoint),
+            (qml.ctrl, qml.ctrl),
+        ],
+    )
+    def test_nested_adjoint_ctrl(self, func1, func2):
+        """Test that nested adjoint and ctrl successfully pass through autograph"""
+
+        # Build the nested operator
+        op = func2(qml.X) if func2 is qml.adjoint else func2(qml.X, control=0)
+        final_op = func1(op) if func1 is qml.adjoint else func1(op, control=1)
```

Comment:
If you use the `partial` change above:
```suggestion
        op = func2(qml.X)
        final_op = func1(op)
```